src/integral.jl

```diff
@@ -3,7 +3,7 @@
 ################################################################################
 
 """
-    integral(f, geometry[, rule]; FP=Float64)
+    integral(f, geometry[, rule]; diff_method, FP)
```

Comment:
```suggestion
    integral(f, geometry[, rule]; diff_method=_default_method(geometry), FP=Float64)
```
Why not specify the default arguments already here?